### ISSUE TITLE
Fails on npm install

### ISSUE THREAD

**Issue by newtriks** [oncletom/Imager.jsx#5]
```
$ npm install imager.jsx
-
> imager.jsx@1.1.0 postinstall /Users/newtriks/src/projects/funk-doctor/node_modules/imager.jsx
> if [[ $NODE_ENV != 'production' ]]; then npm run build; fi


> imager.jsx@1.1.0 build /Users/newtriks/src/projects/funk-doctor/node_modules/imager.jsx
> npm run build-vendor && npm run build-bundle


> imager.jsx@1.1.0 build-vendor /Users/newtriks/src/projects/funk-doctor/node_modules/imager.jsx
> browserify -r react -r lodash.debounce -o dist/vendor.js


events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: ENOENT, open 'dist/vendor.js'

npm ERR! imager.jsx@1.1.0 build-vendor: `browserify -r react -r lodash.debounce -o dist/vendor.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the imager.jsx@1.1.0 build-vendor script.
npm ERR! This is most likely a problem with the imager.jsx package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     browserify -r react -r lodash.debounce -o dist/vendor.js
```

**Comment by oncletom** [comment]
After a quick lookup, it seems the `dist` folder is not packaged with the npm package. Weird.

**Comment by newtriks** [comment]
Hah, I didn't have time to delve sorry, but now you indicated where the problem lies I can see you're ignoring the `.gitignore`, so the `dist` directory is not included as it's seen as empty. Adding `!.gitignore` to the file should fix :)

**Comment by oncletom** [comment]
Published as `v1.1.1`, thank you :-)
